scripts/migrate_registration_logs.py

```diff
@@ -48,4 +87,4 @@ def main(dry_run):
     dry_run = 'dry' in sys.argv
```

Comment:
Minor: it looks like the convention is to use `--dry` instead of `dry` (based on the other scripts in https://openscience.atlassian.net/browse/OSF-5986).